ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/helpers/BeaconStateAccessors.java

```diff
@@ -90,6 +91,22 @@ public Optional<BLSPublicKey> getValidatorPubKey(BeaconState state, UInt64 valid
                 }));
   }
 
+  public boolean validateWithdrawalCredentials(
```

Comment:
I'd have modelled this off of `verifyVoluntaryExits` and return a `BlockValidationResult` so that we can include a reason for it being invalid.

Reply:
Also, why is this here in `BeaconStateAccessors`? I'd have expected this to just be part of `BlockProcessorCapella` since it's actually part of the new `process_bls_to_execution_change` method.